src/EventHub/EventHub/help/Az.EventHub.md

```diff
@@ -60,25 +58,25 @@ Gets lists of resources that supports Privatelinks.
 Gets the details of an EventHub schema group.
 
 ### [New-AzEventHub](New-AzEventHub.md)
-Create a new Event Hub as a nested resource within a Namespace.
+create a new Event Hub as a nested resource within a Namespace.
```

Comment:
The first letter c is not capitalized.

Reply:
Update to this PR forthcoming, but incorrect in [the Synopsis of the reference doc](https://github.com/Azure/azure-powershell/blob/main/src/EventHub/EventHub/help/New-AzEventHub.md#synopsis).